lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -56,6 +58,102 @@ static StringAttr filterUselessName(StringAttr name) {
   return isUselessName(name.getValue()) ? StringAttr() : name;
 }
 
+/// Return a nullable Value that is, optionally, the port that this
+/// value originates from.
+static void getPorts(Value value, SmallVector<Value, 4> &ports) {
```

Comment:
Small suggestion:
```suggestion
static void getPorts(Value value, SmallVectorImpl<Value> &ports) {
```

Reply:
Cool! Didn't know this, but it makes sense... I've included this in the rebase.